src/ncp-spinel/SpinelNCPInstance.cpp

```diff
@@ -1252,6 +1253,9 @@ unpack_thread_network_time(const uint8_t *data_in, spinel_size_t data_len, boost
 			entry.clear();
```

Comment:
I agree with your conclusion here. However, changing this will affect existing users of wpantund who may have written dbus client code to expect this structure.

Example, when running `getprop TimeSync:ThreadNetworkTime:AsValMap` from `wpanctl`:

Before:

```
TimeSync:ThreadNetworkTime:AsValMap = [
	[
		"ThreadNetworkTime" => 0x000000000030AD25
		"TimeSyncReceivedMonoTimeUs" => 0x000000227F5DF606
		"TimeSyncStatus" => 1
	]
]
```

After:
```
TimeSync:ThreadNetworkTime:AsValMap = [
	"ThreadNetworkTime" => 0x000000000305724B
	"TimeSyncReceivedMonoTimeUs" => 0x000000227A871491
	"TimeSyncStatus" => 1
]
```

Reply:
I see your point. I agree, let's keep it as is.